Index: components/apppp/apppp.java
===================================================================
--- components/apppp/apppp.java	(revision a2eb2ffabc63324d3600805c57d759dbc8513747)
+++ components/apppp/apppp.java	(revision a2eb2ffabc63324d3600805c57d759dbc8513747)
@@ -0,0 +1,6 @@
+public class apppp{
+
+    public static void main(String[] args) {
+        System.out.println("test");
+    }
+}
\ No newline at end of file
